docs/manual/index-checker.tex

```diff
@@ -303,23 +303,27 @@ \section{Array minimum lengths\label{index-minlen}}
 annotated as
 \refqualclasswithparams{checker/index/qual}{LTLengthOf}{"arr"}.
 
+The MinLen type system is implemented in practice by the Constant Value Checker
```

Comment:
The Index Checker's manual section should at least include an explanation of how to use `@MinLen` and why a programmer needs it while using the Index Checker. I'm fine with moving the documentation of the annotation itself to the Value Checker's manual section, though.